docs/howto/event-prep.md

```diff
@@ -0,0 +1,133 @@
+# Decide if the infrastructure needs preparation before an Event
+
+A hub's specific setup is usually optimized based on the day to day usage expectations. But because events usually imply a different usage pattern, the infrastructure might need to be adjusted in order to accommodate the spikes in activity.
+
+```{important}
+The communities we serve have the responsibility to notify us about an event they have planned on a 2i2c hub [at least three weeks before](https://docs.2i2c.org/community/events/#notify-the-2i2c-team-about-the-event) the event will start. This should allow us enough time to plan and prepare the infrastructure for the event properly if needed.
+```
+
+The events might vary in type, so the following list is not complete and does not cover all of them (yet). Most common event types are exams, workshops etc.
+
+## Event checklist
+
+Below are listed the main aspects to consider adjusting on a hub to prepare it for an event:
+
+### 1. Quotas
+
+We must ensure that the quotas from the cloud provider are high-enough to handle expected usage. It might be that the number of users attending the event is very big, or their expected resource usage is big, or both. Either way, we need to check the the existing quotas will accommodate the new numbers.
+
+```{admonition} Action to take
+:class: tip
```

Comment:
````suggestion
```{tip} Action to take
````

Does this work?

Reply:
Unfortunately "Tip" is still the name of the admonition. This is how it looks like in both versions:

<img width="814" alt="Screenshot 2023-12-13 at 15 39 20" src="https://github.com/2i2c-org/infrastructure/assets/7579677/c6b8ad45-ee17-4233-9d42-34c9ff0a4132">
<img width="814" alt="Screenshot 2023-12-13 at 15 39 34" src="https://github.com/2i2c-org/infrastructure/assets/7579677/fec1bb38-02f0-413a-9742-c3d9c2d5a592">

Reply:
Ohhhh, sorry, I get what you're trying to do now